Tools/build/umarshal.py

```diff
@@ -178,6 +181,23 @@ def r_object(self) -> Any:
         finally:
             self.level = old_level
 
+    def r_bytecode(self) -> bytes:
+        nbytes = self.r_long() * 2
+        bytecode = bytearray()
+        while len(bytecode) < nbytes:
+            opcode_byte = self.r_byte()
+            if opcode.HAVE_ARGUMENT <= opcode_byte:
```

Comment:
This looks weird. I'm so used to `if x >= N` that seeing `if N <= x` just feels wrong.
```suggestion
            if opcode_byte >= opcode.HAVE_ARGUMENT:
```

Reply:
`const == variable` is a C-ism to prevent accidentally writing `=` instead of `==`. No need in Python.

Reply:
I have a weird habit where I always use `<` and `<=` for comparisons, regardless of the placement of constants, etc. I guess I like the parallel with chained comparisons like `lo <= x < hi`, which almost always use `<` and `<=`.

I'll change it, though.